### PR TITLE
Fix/functional wiki template

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java [hunk 1]
```diff
@@ -0,0 +1,732 @@
...
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.core.PLFData;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiHomePage;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiManagement;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiSettingPage;
+import org.exoplatform.platform.qa.ui.wiki.pageobject.WikiValidattions;
+
+@Tag("functional")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java]: Is it a new type of tests ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+   * are listed
+   */
+  @Test
+  public void test06_SearchTemplateWhenSearchFieldIsBlank() {
+    info("Test 6: Search template when Search field is blank");
+
+    info("Go to Wiki Settings");
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToWikiSettingPage();
+    ArrayList<String> arrayTemplate = new ArrayList<String>();
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java]: Always use the interface when possible, here it should be : List<String> arrayTemplate = new ArrayList<String>();

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java]: Yes, it is new type. We have 3 types: smoke, sniff and functional

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java (hunk 1 above)
Comment (thomasdelhomenie) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/functional/WikiTemplateTestIT.java]: ok :+1:

**Pull request merged**